token.sh

```diff
@@ -14,7 +15,8 @@ API_HEADER="Accept: application/vnd.github.${API_VERSION}+json"
 AUTH_HEADER="Authorization: token ${ACCESS_TOKEN}"
 CONTENT_LENGTH_HEADER="Content-Length: 0"
 
-case ${RUNNER_SCOPE} in
+
+case ${RUNNER_SCOPE:-} in
```

Comment:
If theres a reason for it to be empty (which I think this handles) lets be explicit since its a new case. I dont think its valid to be empty, so it would cause new behavior here to fall into the `*)`